cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -65,9 +74,58 @@ describe('Create scenario', () => {
 
     // Delete all tests scenarios
     ScenarioManager.switchToScenarioManager();
-    for (const scenarioName of scenarioNamesToDelete) {
-      ScenarioManager.deleteScenario(scenarioName);
-    }
+    scenarioNamesToDelete.forEach((scenarioName) => {
+      if (Array.isArray(scenarioName)) {
+        scenarioName.forEach((scenarioNameFromArray) => {
+          ScenarioManager.deleteScenario(scenarioNameFromArray);
+        });
+      } else {
+        ScenarioManager.deleteScenario(scenarioName);
+      }
+    });
+  });
+
+  it('confirmation launch dialog can be hide', () => {
```

Comment:
These tests are probably out of scope of the "CreateScenario" test file. Could we move the tests for the "don't ask again" pop-up in a new test file ?